cmdconfig/builder.go

```diff
@@ -18,6 +21,7 @@ func OnCmd(cmd *cobra.Command) *CmdBuilder {
 	cfg.bindings = map[string]*pflag.Flag{}
 
 	originalPreRun := cfg.cmd.PreRun
+	originalRun := cfg.cmd.Run
```

Comment:
add comment to explain why we need to do this

Reply:
comments added